### ISSUE TITLE
Landmark roles visual helper does not show accessible name set by aria-labelledby

### ISSUE THREAD

**Issue by mrwweb** [microsoft/accessibility-insights-web#7761]
## Describe the bug

When using `aria-labelledby` to give a landmark an accessible name, I'd expect it to appear in the "Visual helper" for the "Landmark roles" check. An accessible name set with `aria-label` DOES appear, as expected, so I'd expect `aria-labelledby` to work similarly given that it's [Example 2 of the sufficient techniques for labelling regions](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11#example-2-multiple-landmarks-of-the-same-type-and-aria-labelledby) from WCAG.

**To Reproduce**

1. View pen
2. Run a full Assessment with Accessibility Insights
3. Go to check 4.1, "Landmark roles"
4. Turn on Visual helper toggle
5. observe missing label for navigation landmark with `aria-labelledby`

## HTML repro

[Code pen demo](https://codepen.io/editor/mrwweb/pen/019e8f9f-4fe6-7c5f-8f29-b4223bda97d1)

```html
<nav aria-labelledby="nav-heading">
    <h2 class="screen-reader-text" id="nav-heading">Accessible Name for Nav</h2>
    <ul>
        <li><a href="#">Link</a></li>
        <li><a href="#">Link</a></li>
        <li><a href="#">Link</a></li>
    </ul>
</nav>
```

## Expected behavior

Accessible name set by `aria-labelledby` is visible in the Accessibility Insights Visual helper for the Landmark roles check

## Screenshots

Screenshot of example with visual helper enabled:

<img width="441" height="82" alt="demo html with visual helper show no visible label before 'navigation LM' label" src="https://github.com/user-attachments/assets/274e465c-bcc4-4b24-91a2-25c28ac35b0b" />

## Context (please complete the following information)

- OS Name & Version: Windows 11
- Browser Version: Edge Version 148.0.3967.96 (Official build) (64-bit)

## Are you willing to submit a PR?

Maybe?

## Did you search for similar existing issues?

Yup!
